Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.cpp

```diff
@@ -343,44 +351,127 @@ namespace Terrain
         AZ::Vector3 mainCameraPosition = mainView->GetCameraTransform().GetTranslation();
         CheckStacksForUpdate(mainCameraPosition);
 
-        for (auto& view : process.m_views)
+        // First pass - gather a list of all sectors that could render based on their status and lod, disregarding camera frustums.
+
+        const float minMovedDistanceSq = m_sampleSpacing * m_sampleSpacing; // This could probably be relaxed further, but is a good starting point.
+        if (m_previousCameraPosition.GetDistanceSq(mainCameraPosition) > minMovedDistanceSq)
         {
-            float minDistanceSq = 0.0f;
+            m_previousCameraPosition = mainCameraPosition;
+
             float maxDistanceSq = m_config.m_firstLodDistance * m_config.m_firstLodDistance;
+            uint32_t nextLodSectorCount = m_1dSectorCount * 2; // The number of this lod's sectors that would fit into the next lod's space.
+            AZStd::vector<bool> previousSelectedSectors;
 
-            const AZ::Frustum viewFrustum = AZ::Frustum::CreateFromMatrixColumnMajor(view->GetWorldToClipMatrix());
-            const AZ::Vector3 viewVector = viewFrustum.GetPlane(AZ::Frustum::PlaneId::Near).GetNormal();
-            const AZ::Vector3 viewPosition = view->GetCameraTransform().GetTranslation();
+            m_sectorsToDraw.clear();
 
-            for (auto& sectorStack : m_sectorStack)
+            for (uint32_t stackIndex = 0; stackIndex < m_sectorStack.size(); ++stackIndex)
             {
-                float radius = 0.0f;
-                [[maybe_unused]] AZ::Vector3 center;
-                sectorStack.m_sectors.at(0).m_aabb.GetAsSphere(center, radius);
+                auto& sectorStack = m_sectorStack.at(stackIndex);
 
-                for (auto& sector : sectorStack.m_sectors)
+                // Construct a grid of bools that covers the entire space of the next LOD at the scale of the current LOD, and
+                // mark everything to false. As sectors in this LOD are drawn, mark appropriate locations in the grid as true.
+                // The next LOD will use this data to figure out where it shouldn't draw because a higher lod has already drawn.
+
```

Comment:
This might benefit from a little more description. Maybe something like this?
```
Each sector in an LOD is 1/2 the size of a sector in the next LOD in each direction, so 4 sectors
in this LOD equal one sector in the next LOD. Construct a grid of bools for each sector in this
LOD that covers the entire space of the next LOD, and mark everything to false. As sectors in
this LOD are drawn, mark appropriate locations in the grid as true. When processing the next
LOD, each of the next LOD's sectors will look up the 4 entries in this that represent quadrants
of that sector to determine whether to draw nothing, specific quadrants, or the entire sector.
```